src/server.ts

```diff
@@ -135,13 +135,16 @@ async function startApolloServer() {
     resolvers,
     introspection: true,
     allowBatchedHttpRequests: true,
+    includeStacktraceInErrorResponses: true,
```

Comment:
Tror vi må enable denne for å få tilgang på stacktracer der vi logger så jeg tror vi manuelt må fjerne den :smile:

Reply:
Er det det du har gjort rett under? Isåfall så det jo egentlig ikke så ille ut som det hørtes ut som